modules/katello_content_view.py

```diff
@@ -164,7 +162,8 @@ def main():
     )
 
     if not HAS_NAILGUN_PACKAGE:
-        module.fail_json(msg="Missing required nailgun module (check docs or install with: pip install nailgun")
+        module.fail_json(msg="Missing required nailgun module (check docs or install with: "
+                         "pip install nailgun")
```

Comment:
Ah, found it. https://github.com/ansible/proposals/issues/50 Yeah, I'll update this. Looks like they're using `pep8 --ignore=E402 --max-line-length=160`